app/activity_notifications/message_received.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# To deliver this notification:
+#
+# MessageReceived.with(contributor: @contributor, request: @request).deliver_later(current_user)
+# MessageReceived.with(contributor: @contributor, request: @request).deliver(current_user)
+
+class MessageReceived < Noticed::Base
+  # Add your delivery methods
+  #
+  deliver_by :database
+  # deliver_by :email, mailer: "UserMailer"
+  # deliver_by :slack
+  # deliver_by :custom, class: "MyDeliveryMethod"
+
+  # Add required params
+  #
+  param :contributor, :request
+
+  # Define helper methods to make rendering easier.
+  #
+  # rubocop:disable Rails/OutputSafety
+  def message
+    t('.message_html',
+      contributor_name: contributor.name,
+      request_title: params[:request].title).html_safe
+  end
+  # rubocop:enable Rails/OutputSafety
```

Comment:
Don't really like this. I have seen from the rails docs:

> [Keys with a '_html' suffix and keys named 'html' are marked as HTML safe. When you use them in views the HTML will not be escaped.](https://api.rubyonrails.org/v6.1.5/classes/ActionView/Helpers/TranslationHelper.html#method-i-translate)

But, when I've attempted to do this, it just displays the `<b>` as plain text.